### ISSUE TITLE
[feature request] display the last N modifications of the repo

### ISSUE THREAD

**Issue by goofy-bz** [mozfr/transvision#737]
The idea is to make easier the review of the last commits on the locale repository.
Depending on what is technically possible, it could be
- show the list of the last 50/100 editions
- show the list of all editions since N hours/days
thus the QA reviewer could have a single updated list to browse ;-)

**Comment by TheoChevalier** [comment]
We’re thinking about listing new strings per release, would that cover this need?

Basically, this would be similar to this view, but you’ll be able to go through all releases:
https://transvision.mozfr.org/channelcomparison/?locale=fr&chan1=central&chan2=aurora#new_strings -> those are the new strings for Firefox, Fennec, SeaMonkey, Thunderbird that have landed during Gecko 48 cycle.

**Comment by goofy-bz** [comment]
> We’re thinking about listing new strings per release, would that cover this need?


Yes it would for most of the needs. But it would not cover intermediate "daily" mods such as http://hg.mozilla.org/l10n-central/fr/rev/59b190e239a6.   

Ideally, the list should include __every app and every channel__ + have the "report a bug" feature as in usual result page.  

Usecase: I push the button to get *all* modified and new items since 3 days in a list and can report if any flaw.

But I suppose everything is not that easy to implement. Thanks for pointing me to the "#new_strings" feature :)

**Comment by flodolo** [comment]
Sadly I don't think it's doable. 

We only store the current strings, we don't store any history. Comparing two channels at the same moment is easy, to add something like this we would need to store not only the current state, but also the changes, and when they happened. Then multiply that for 4 branches and ~100 locales. 

It would be a huge amount of data, and a completely different storage model from what we currently is.

**Comment by goofy-bz** [comment]
I understand :)  

I hope anyway you can make the new_strings feature available with some improvements. TIA

**Comment by TheoChevalier** [comment]
This new Pontoon feature should actually cover the needs (it’s in the search panel). Firefox is not yet set up on Pontoon for French, but this will be the case in the future. This will also be handy to suggest a fix, or fix the mistake right away.

<img width="349" alt="capture d ecran 2016-07-31 a 06 24 14" src="https://cloud.githubusercontent.com/assets/1294206/17274567/9bd5ebae-56e7-11e6-979c-263218200ce7.png">
